frugalos_segment/src/synchronizer.rs

```diff
@@ -80,6 +82,16 @@ impl Synchronizer {
                 .label("type", "delete")
                 .finish()
                 .unwrap(),
+            repairs_success_total: metric_builder
+                .counter("repairs_success_total")
+                .label("type", "repair")
+                .finish()
+                .unwrap(),
```

Comment:
note: one should use `?` for error handling, or `expect` to express the intention. Why is `unwrap` used here?

Reply:
Either's fine.

Reply:
Replaced all occurrences (including here) with `expect` because they should not fail.